File: ObjC_Example/Fullboard/FullBoardAdContentViewController.m
//
//  FullBoardAdContentViewController.m
//  ObjC_Example
//
//  Copyright © 2017年 FAN Communications. All rights reserved.
//

#import "FullBoardAdContentViewController.h"

@interface FullBoardAdContentViewController ()

@property (weak, nonatomic) IBOutlet UILabel *pageLabel;

@end

@implementation FullBoardAdContentViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
    // Do any additional setup after loading the view from its nib.
    
    _pageLabel.text = self.number;
}

@end

File: Swift_Example/Fullboard/FullBoardAdContentViewController.swift
//
//  FullBoardAdContentViewController.swift
//  ObjC_Example
//
//  Copyright © 2017年 FAN Communications. All rights reserved.
//

import UIKit

class FullBoardAdContentViewController: UIViewController {
    
    @IBOutlet fileprivate weak var pageLabel: UILabel!
    var number: String?
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        self.pageLabel.text = number;
    }
}
